### ISSUE TITLE
Website documentation on longitudinal model

### ISSUE THREAD

**Issue by contsili** [predictive-clinical-neuroscience/PCNtoolkit#458]

**Comment by contsili** [comment]
### Longitudinal normative modelling (LNM): how would the users run their analysis

**Prerequisite**: we expect the user has some longitudinal data themselves. Otherwise the LNM analysis does not make sense to do

**Important - all the models are cross sectional**:
Even when trained on longitudinal data, the model treats every visit as an independent measurement and does not know that two rows come from the same person. For this reason a pretrained model used for LNM analysis does not need only longitudinal subjects, it can be entirely cross-sectional, entirely longitudinal, or a mix. 

The longitudinal analysis steps are listed below:

1.	**Normative model**

    Either fit a new model on your data, or transfer/extend a pretrained model to your local data. This is unchanged from any existing non-longitudinal analysis.

2.	**Longitudinal modelling**

    There are two options:

    a.	Load precomputed correlation matrix

    Without a normative model users can probably only compute z-gain scores on their own longitudinal data, but CANNOT place thrivelines on a centile plot. So the advice is to have a normative model from step 1.

    The users don't need the same normative model as the one the correlation matrices were computed with, they can use any model. The thrivelines are computed from the correlation matrix, and then every normative model can map them into its own y space/ its own centiles.

    b.	Compute correlation matrix

    A normative model from step 1. is a must have here as it gives the required z-scores to compute the correlation matrix.

    Users need longitudinal subjects in the order of thousand  to be able to compute a meaningful correlation matrix

     Example: Bayer et al., 2026 had ~ 10,500 subjects, contributing ~22,500 scans (up to 8 timepoints each) + she put them in the test set (not train set)

**Comment by likeajumprope** [comment]
> ### Longitudinal normative modelling (LNM): how would the users run their analysis
> **Prerequisite**: we expect the user has some longitudinal data themselves. Otherwise the LNM analysis does not make sense to do
> 
> **Important - all the models are cross sectional**: Even when trained on longitudinal data, the model treats every visit as an independent measurement and does not know that two rows come from the same person. For this reason a pretrained model used for LNM analysis does not need only longitudinal subjects, it can be entirely cross-sectional, entirely longitudinal, or a mix.

I do not quite understand the part above - the longitudinal models do treat the data as longitudinal?
> 
> The longitudinal analysis steps are listed below:
> 
> 1. **Normative model**
>    Either fit a new model on your data, or transfer/extend a pretrained model to your local data. This is unchanged from any existing non-longitudinal analysis.

I am also not so sure whether i understand the statement above

> 2. **Longitudinal modelling**
>    There are two options:
>    a.	Load precomputed correlation matrix
>    Without a normative model users can probably only compute z-gain scores on their own longitudinal data, but CANNOT place thrivelines on a centile plot. So the advice is to have a normative model from step 1.

I think we have to be really clear what we mean with the terminology here. without a cross-sectional normative model?

>    The users don't need the same normative model as the one the correlation matrices were computed with, they can use any model. The thrivelines are computed from the correlation matrix, and then every normative model can map them into its own y space/ its own centiles.


>    b.	Compute correlation matrix
>    A normative model from step 1. is a must have here as it gives the required z-scores to compute the correlation matrix.
>    Users need longitudinal subjects in the order of thousand  to be able to compute a meaningful correlation matrix
>    Example: Bayer et al., 2026 had ~ 10,500 subjects, contributing ~22,500 scans (up to 8 timepoints each) + she put them in the test set (not train set)

I think overall you mean the right things, but I think we need to be really careful with the terminology

**Comment by likeajumprope** [comment]
I would frame it a bit differently:

2 Options: 
1. Use pre-trained longitudinal models.
- transfer your longitudinal data into z-space: get z-scores for longitudinal data under cross-sectional normative model. Ideally, the longitudinal data was preprocessed using longitudinal freesufer
- calculate thivelines and z -gains for each of the longitudinal data points, plot them
- potentially plot data points into thrivelines plot

2. Derive your own correlation matrices:
- what you wrote

**Comment by contsili** [comment]
when we release the tutorial, it would be good to get feedback from @bbuckova too

**Comment by contsili** [comment]
we want two tutorials: 1 for loading a correlation matrix, 1 for computing one
